client/components/domains/search-filters/style.scss

```diff
@@ -4,7 +4,6 @@
 .search-filters__dropdown-filters {
 	align-items: center;
 	height: 40px;
-	z-index: z-index("root", ".search");
```

Comment:
I don't think we need this anymore. Removing it yielded no changes in the production version.